utils/src/com/cloud/utils/S3Utils.java

```diff
@@ -352,10 +352,15 @@ public static File getFile(final ClientOptions clientOptions, final String bucke
         ListObjectsRequest listObjectsRequest = new ListObjectsRequest().withBucketName(bucketName).withPrefix(directory + SEPARATOR);
 
         ObjectListing ol = client.listObjects(listObjectsRequest);
```

Comment:
Will improve it when I update the entire S3 implementation to the latest AWS sdk version, will be in the next month.